src/proposals/20201128/SparkEthereum_20241128.t.sol

```diff
@@ -0,0 +1,218 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.25;
+
+import { IERC4626 } from 'forge-std/interfaces/IERC4626.sol';
+import { IERC20 }   from 'forge-std/interfaces/IERC20.sol';
+import { Address }  from 'src/libraries/Address.sol';
+
+import { SparkEthereumTestBase, ReserveConfig, MarketParams, Ethereum, IMetaMorpho } from 'src/SparkTestBase.sol';
+import { Base }                                                                      from 'spark-address-registry/Base.sol';
+
+import { Bridge }                from "xchain-helpers/testing/Bridge.sol";
+import { Domain, DomainHelpers } from "xchain-helpers/testing/Domain.sol";
+import { OptimismBridgeTesting } from "xchain-helpers/testing/bridges/OptimismBridgeTesting.sol";
+import { StdChains }             from "forge-std/StdChains.sol";
+
+import { ForeignController } from 'spark-alm-controller/src/ForeignController.sol';
+import { IExecutor }         from 'spark-gov-relay/src/interfaces/IExecutor.sol';
+
+interface DssAutoLineLike {
+  function setIlk(bytes32 ilk, uint256 line, uint256 gap, uint256 ttl) external;
+  function exec(bytes32 ilk) external;
+}
+
+contract SparkEthereum_20241128Test is SparkEthereumTestBase {
+    using DomainHelpers         for StdChains.Chain;
+    using DomainHelpers         for Domain;
+
+    address internal constant PT_27MAR2025_PRICE_FEED = 0x38d130cEe60CDa080A3b3aC94C79c34B6Fc919A7;
+    address internal constant PT_SUSDE_27MAR2025      = 0xE00bd3Df25fb187d6ABBB620b3dfd19839947b81;
+    address internal constant PT_USDE_27MAR2025       = 0x8A47b431A7D947c6a3ED6E42d501803615a97EAa;
+
+    address internal constant AUTO_LINE     = 0xC7Bdd1F2B16447dcf3dE045C4a039A60EC2f0ba3;
```

Comment:
Where did you get it from? just curious

Reply:
@hexonaut provided it, finding it on my own could've taken a while :grin:

Reply:
Was it in DM? Just couldn't find it on Slack

Reply:
Sky addresses are here: https://chainlog.sky.money/api/mainnet/active.json